code/game/machinery/computer/arcade.dm

```diff
@@ -571,23 +571,24 @@
 		event = null
 
 	else if(href_list["killcrew"]) //shoot a crewmember
-		var/sheriff = remove_crewmember() //I shot the sheriff
-		playsound(loc,'sound/weapons/gunshots/gunshot.ogg', 100, 1)
+		if(settlers.len > 0 || alive > 0)
+			var/sheriff = remove_crewmember() //I shot the sheriff
+			playsound(loc,'sound/weapons/gunshots/gunshot.ogg', 100, 1)
```

Comment:
spacing here between the parameters

Reply:
Can you clarify what you mean here? Should it something like `settlers.len>0 || alive>0` or `settlers.len > 0||alive > 0`?

Reply:
the if statement itself is fine. I meant the `playsound(loc,'sound....)`. Standard is to put a space after each `,` or operator and the like

Reply:
Ah I understand, thanks!